holoviews/plotting/bokeh/plot.py

```diff
@@ -235,24 +263,30 @@ def _create_subplots(self, layout, ranges):
             # Create axes
             kwargs = {}
             if c == 0 and r != 0:
-                kwargs['xaxis'] = 'bottom-bare'
-                kwargs['width'] = 150
+                kwargs['xaxis'] = None
+                kwargs['width'] = self.plot_size+50
             if c != 0 and r == 0:
```

Comment:
True, it's the offset used to account for the axis width. Could parameterize it I suppose.